src/test/java/api/loans/LoanAPITests.java

```diff
@@ -1938,6 +1938,71 @@ private void queryLoans(int limit)
     });
   }
 
+  @Test
+  public void canGetLoanWithPatronGroupAfterAnonymize() throws InterruptedException,
```

Comment:
I think this test could be used more generally for anonymized loans

```suggestion
  public void canGetAnonymizedLoan() throws InterruptedException,
```

Reply:
Ok, I will do it